utils/mil_tools/nodes/online_bagger.py

```diff
@@ -346,7 +359,7 @@ def start_bagging(self, req):
             # get time index the normal way
             else:
                 bag_index = self.get_time_index(topic, requested_seconds)
-
+            rospy.loginfo('topic: {}, bag_index: {}'.format(topic, bag_index))
```

Comment:
If I'm not familiar with this script's code, I probably won't know what bag_index is. Try to format logging so that the average Joe Smoe would get something useful from it, like the famous line 261.